expirationd.lua

```diff
@@ -51,7 +51,9 @@ local constants = {
     -- default function for process_while
     process_while = function(...) return true end,
     -- default iterating over the loop will go in ascending index
-    iterator_type = "ALL"
+    iterator_type = "ALL",
+    -- default atomic_iteration is false, batch of items doesn't include in one transaction
+    atomic_iteration = false
```

Comment:
nit: Please add a comma at the end of line. It will help to make diff a less if you plan to add some new option. Maybe it's also related for previous commits.

Reply:
Fixed for all commits